src/app/CMakeLists.txt

```diff
@@ -446,6 +446,7 @@ target_link_libraries(app-lib
   ui-lib
   undo-lib
   ${TINYXML_LIBRARY}
+  ${CURL_LIBRARIES}
```

Comment:
This is the only change I think it's not needed (because `net-lib` already depends on `${CURL_LIBRARIES}` in the next commit). I'll merge all the changes and revert this commit to see what happens. Please tell me if it's needed again for some specific scenario.